posthog/api/decide.py

```diff
@@ -231,12 +231,8 @@ def get_decide(request: HttpRequest):
                         response["analytics"] = {"endpoint": settings.NEW_ANALYTICS_CAPTURE_ENDPOINT}
 
             if (
-                settings.ELEMENT_CHAIN_AS_STRING_TEAMS
-                and str(team.id) in settings.ELEMENT_CHAIN_AS_STRING_TEAMS
-                and (
-                    settings.ELEMENT_CHAIN_AS_STRING_EXCLUDED_TEAMS is None
-                    or str(team.id) not in settings.ELEMENT_CHAIN_AS_STRING_EXCLUDED_TEAMS
-                )
+                settings.ELEMENT_CHAIN_AS_STRING_EXCLUDED_TEAMS
```

Comment:
nit: giving an empty `ELEMENT_CHAIN_AS_STRING_EXCLUDED_TEAMS` would not release it for everyone, but disable it for everyone, which feels counter-intuitive. I'd remove this check, unless that's your intent